test/unit/live-tracker.test.js

```diff
@@ -169,15 +154,15 @@ QUnit.module('LiveTracker', () => {
   QUnit.test('single seekable, helpers should be correct', function(assert) {
     // simple
     this.player.seekable = () => createTimeRanges(10, 50);
-    assert.strictEqual(this.liveTracker.liveWindow(), 40, 'liveWindow is 40s');
+    assert.strictEqual(this.liveTracker.liveWindow(), 40.03, 'liveWindow is 40s');
```

Comment:
Why the .03?

Reply:
`liveWindow()` now uses `liveCurrentTime` instead of `seekableEnd`. We increment `live CurrentTime` by 0.03 every 30ms, hence `.03`

Reply:
oh and we start at 0.03 ahead at init, since we are falling behind every ms the player isn't ready.